### DISCUSSION TITLE
The Mojo development environment was not found.

### DISCUSSION THREAD

**Discussion by o2candoo** [modular/modular#988] [Q&A]
I just installed ubuntu on my windows 11 machine, installed modular, mojo etc. as it is described on the homepage. However, after starting vs code through 'code .', i open a script called 'main.mojo', try to run it, then vs-code is providing me this message:

The Mojo🔥 development environment was not found. If the Mojo SDK is installed, please set the MODULAR_HOME environment variable to the appropriate path, or set the `mojo.modularHomePath` configuration. If you do not have it installed, would you like to install it?

Which is the appropriate path? Any suggestions?

**Comment by Sharktheone** [comment]
I belive you need to export `$MODULAR_HOME` to `$HOME/.modular` 

You can also read about this topic in #940
